ballerina/tests/test.bal

```diff
@@ -36,7 +38,30 @@ ConnectionConfig sfConfig = {
     }
 };
 
-Client baseClient = check new (sfConfig);
+ConnectionConfig sfConfigPasswordFlow = {
+    baseUrl: baseUrl,
+    auth: {
+        password,
+        username,
+        tokenUrl: refreshUrl,
+        clientId: clientId,
+        clientSecret: clientSecret,
+        credentialBearer: "POST_BODY_BEARER"
+    }
+};
+
+ConnectionConfig sfConfigCredentialsFlow = {
+    baseUrl: baseUrl,
+    auth: {
+        clientId: clientId,
+        clientSecret: clientSecret,
+        tokenUrl: refreshUrl
+    }
+};
```

Comment:
Shall we have the suffix `Config` for all the config variables in here, as per our usual conventions?
we can use variable names like `sfBearerTokenConfig`, `sfClientCredentialsConfig`, etc.

Reply:
https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/308